### PR TITLE
Improve error message when `Pulumi.yaml` can't be found

### PR DESCRIPTION
**Pull request by ellismg**

### CODE REVIEW

File: cmd/util.go [hunk 1]
```diff
@@ -286,7 +286,8 @@ func readProject() (*workspace.Project, string, error) {
 			"could not locate Pulumi.yaml project file (searching upwards from %s)", pwd)
 	} else if path == "" {
 		return nil, "", errors.Errorf(
-			"no Pulumi.yaml project file found (searching upwards from %s)", pwd)
+			"no Pulumi.yaml project file found (searching upwards from %s). If you have not "+
+				"created project yet, try `pulumi new`", pwd)
```
Comment (pgavlin) [inline cmd/util.go]: nit: "created a project yet, use `pulumi new` to do so" + same below

Comment (ellismg) [reply cmd/util.go]: SGTM.  Done.

**Comment by ellismg** [comment]
@pgavlin PTAL

File: cmd/util.go [hunk 2]
```diff
@@ -286,7 +286,8 @@ func readProject() (*workspace.Project, string, error) {
 			"could not locate Pulumi.yaml project file (searching upwards from %s)", pwd)
 	} else if path == "" {
 		return nil, "", errors.Errorf(
-			"no Pulumi.yaml project file found (searching upwards from %s)", pwd)
+			"no Pulumi.yaml project file found (searching upwards from %s). If you have not "+
+				"created project yet, use `pulumi new` to do so", pwd)
```
Comment (pgavlin) [inline cmd/util.go]: `created a project`

**Pull request merged**